src/js/game/Entities/Boat.js

```diff
@@ -30,5 +30,26 @@ module.exports = class Boat extends BaseEntity {
     // Initialize.
     this.sprite.x = this.offset[0] + 40 * this.position.x;
     this.sprite.y = this.offset[1] + 40 * this.position.y;
+
+    Boat.addBobTween(this.game, this.sprite);
+  }
+
+  /**
+   * Apply a "bob up and down in the water" animation to the sprite,
+   * which runs forever.
+   * @param {Phaser.Game} game
+   * @param {Phaser.Sprite} sprite
+   * @returns {Phaser.Tween}
+   */
+  static addBobTween(game, sprite) {
+    return game.add.tween(sprite).to(
+      { y: '-3' },
+      1000,
+      Phaser.Easing.Quadratic.InOut,
```

Comment:
Nit: sinusoidal?

Reply:
Are you saying you'd prefer `Phaser.Easing.Sinusoidal.InOut` to `Phaser.Easing.Quadratic.InOut`?  Any particular reason?  I'm cool with either one, the difference is pretty subtle.

Reply:
My thought was that sinusoidal might be closer to wave action, butI think either is fine!